ankdown/ankdown.py

```diff
@@ -261,24 +259,31 @@ def __getitem__(self, deckname):
 def field_to_html(field):
     """Need to extract the math in brackets so that it doesn't get markdowned.
     If math is separated with dollar sign it is converted to brackets."""
-    if CONFIG['dollar']:
+    if CONFIG['math_mode'] is 'dollars':
```

Comment:
I think you want `==`, not `is`

Reply:
oh... coffeescript sneaks in ;-) I've pushed a fix